### PR TITLE
translations PR 20180813

### PR DESCRIPTION
**Pull request by enahum**
Ideally this PR should be review by community members for each language.

Note: In the event that you see an english translation instead of the desired language is because this PR was submitted before the translation was made on the Mattermost Translation Server and probably will be submitted for the next PR.

Webapp PR: https://github.com/mattermost/mattermost-webapp/pull/1561

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks all! @enahum looks like the build fails, I tried re-triggering it to see if it was a random unit test failure.

**Comment by enahum** [comment]
Is seems that french has an issue, checking now

**Comment by enahum** [comment]
it should be fixed now

File: i18n/fr.json [hunk 1]
```diff
@@ -2184,7 +2184,7 @@
   },
   {
     "id": "api.user.saml.invalid_extension",
-    "translation": "L'extension ayant l'extension_id={{. ExtensionId}} n'est pas supportée."
+    "translation": "L'extension ayant l'extension_id={{.ExtensionId}} n'est pas supportée."
```
Comment (jasonblais) [inline i18n/fr.json]: FYI @wget ^

Comment (wget) [reply i18n/fr.json]: Ok thanks. Very sorry for the issue since the CI failing was due to French. :-/

**Pull request merged**